types/video.js/index.d.ts

```diff
@@ -4858,6 +4898,13 @@ declare namespace videojs {
          */
         dispose(): void;
 
+        /**
+         * Returns the HTML Video Element
+         *
+         * @return the HTML Video Element
+         */
+        el: () => HTMLVideoElement;
```

Comment:
this could also be an HTMLAudioElement I believe

Reply:
you're right!